paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/cvcrecollection/CvcRecollectionViewModel.kt

```diff
@@ -4,33 +4,24 @@ import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.viewModelScope
 import androidx.lifecycle.viewmodel.CreationExtras
-import com.stripe.android.model.CardBrand
 import kotlinx.coroutines.flow.MutableSharedFlow
-import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.SharedFlow
-import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.asSharedFlow
-import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.launch
 
-internal class CvcRecollectionViewModel(args: Args) : ViewModel() {
+internal class CvcRecollectionViewModel(
+    private val interactor: CvcRecollectionInteractor
+) : ViewModel(), CvcRecollectionInteractor by interactor {
     private val _result = MutableSharedFlow<CvcRecollectionResult>()
-    val result: SharedFlow<CvcRecollectionResult> = _result.asSharedFlow()
 
-    private val _viewState = MutableStateFlow(
-        CvcRecollectionViewState(
-            cardBrand = args.cardBrand,
-            lastFour = args.lastFour,
-            cvc = null,
-            isLiveMode = args.isLiveMode
-        )
-    )
-    val viewState: StateFlow<CvcRecollectionViewState> = _viewState.asStateFlow()
+    // This is used in DisplayMode.Activity to send confirmation or cancellation events
```

Comment:
I don't think this display mode-specific comment is accurate anymore, can you update or remove it?